--- exaile_0.2.5/mmkeys/Makefile~	2006-11-01 01:11:34.000000000 +0100
+++ exaile_0.2.5/mmkeys/Makefile	2006-11-12 19:25:11.200111000 +0100
@@ -1,6 +1,6 @@
 # Copyright 2004 Joe Wreschnig. Released under the terms of the GNU GPL.
 
-PYTHON_DIR = /usr/include/python2.4
+PYTHON_DIR = /usr/include/python2.5
 
 CFLAGS += -fPIC -O2 `pkg-config --cflags gtk+-2.0 pygtk-2.0` -I$(PYTHON_DIR)
 LDFLAGS += `pkg-config --libs gtk+-2.0 pygtk-2.0`
